Install Vault only for single K8s master setup [#1754] (#1755)
epiphany-platform/epiphany: docs/home/howto/SECURITY.md

At the moment only installation on Kubernetes Master is supported, but we are also planning separate installation with no
other components. Also at this moment we are not providing clustered option for Vault deployment, but this will be part

> of the future releases. In Multi Kubernetes Master setup Vault installation is skipped.

Reviewer: ```suggestion
of the future releases. For multi-master (HA) Kubernetes, Vault is not installed.
```
